tests/test_clone_from_s3.py

```diff
@@ -0,0 +1,54 @@
+import tempfile
+import pytest
+import pandas as pd
+from pathlib import Path
+
+from setup_v0_3_study import setup_v0_3_study
+
+from teehr.fetching.s3.clone_from_s3 import list_s3_evaluations, clone_from_s3
+
+def test_get_s3_evaluations_dataframe():
+    """Test get_s3_evaluations as a dataframe."""
+    df = list_s3_evaluations()
+    assert len(df) == 4
+    assert isinstance(df, pd.DataFrame)
+
+def test_get_s3_evaluations_list():
+    """Test get_s3_evaluations as a list."""
+    l = list_s3_evaluations(format="list")
+    assert len(l) == 4
+    assert isinstance(l, list)
+
+
+def test_clone_example_form_s3(tmpdir):
+    """Test filter string."""
+    ev = setup_v0_3_study(tmpdir)
+    clone_from_s3(ev, "p0_2_location_example")
```

Comment:
Just double-checking, here `clone_from_s3` is over-writing the evaluation created by `setup_v0_3_study`, is that correct?  Is it necessary to set up a study first or could the user just do `ev = Evaluation(dir_path=tmpdir)` then clone?

Reply:
That's a good question.  Currently, yes, it is necessary to  first clone the template which is the mostly overwritten.  The main reason for this is to create the folder structure. It would probably be cleaner if it was created automatically behind the scenes.  I'll do that.

Reply:
ok, changed this to be `ev.clone_from_s3()` and made it create the directory structure so we don't need to `ev.clone_template()` first.  I think this is better.  Thanks for the comment.